src/test/components/DepositWidget.test.ts

```diff
@@ -0,0 +1,11 @@
+import { getButtonText, ButtonTypes } from '../../components/DepositWidget/Row'
+
+test('Deposit button text', () => {
```

Comment:
All this one's test the same function, consider using scoping for grouping together:

For example:
```js
describe('Get text for buttons', () => {
  test('Deposit', () => {
    expect(getButtonText(ButtonTypes.DEPOSIT, '')).toMatch(/Deposit/)
  })
  test('Withdraw', () => {
    expect(getButtonText(ButtonTypes.WITHDRAW, '')).toMatch(/Withdraw/)
  })
  test('Enable', () => {
    expect(getButtonText(ButtonTypes.ENABLE, 'TTT')).toMatch(/Enable TTT/)
  })
})
```

![image](https://user-images.githubusercontent.com/2352112/64628065-0a9ab180-d3f1-11e9-946b-b261b9b571c3.png)

Reply:
Thanks for the suggestion, will update.